packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -283,6 +285,10 @@ export async function getBookings({
         },
       },
     },
+    assignmentReason: {
+      orderBy: { createdAt: PrismaClientType.SortOrder.desc },
```

Comment:
I passed "desc" here but I was getting a type error. Although `PrismaClientType.SortOrder.desc` equals "desc"

Reply:
My guess is that when you passed "desc", it considered it just as a string. (Just saying without actually trying it) Maybe you can have

```
const bookingSelect: Prisma.BookingWhereInput = {
...
```

then it might help?

Reply:
Adding `"desc" as const"` would also help in here.